src/core/system_parameters/BEIDOU_B2a_CA.h

```diff
@@ -0,0 +1,529 @@
+const double BEIDOU_MOON_OMEGA_0 = 0.004523601514852e3;      //!< (259 deg 10 min 59.79 sec) [rad]
+const double BEIDOU_MOON_OMEGA_1 = -0.033757146246552e3;     //!< (-1934 deg 08 min 31.23 sec) [rad]
+const double BEIDOU_MOON_GM = 4902.835;                      //!< Lunar gravitational constant [km^3/s^2]
+const double BEIDOU_MOON_SEMI_MAJOR_AXIS = 3.84385243e5;     //!< Semi-major axis of lunar orbit [km];
+const double BEIDOU_MOON_ECCENTRICITY = 0.054900489;         //!< Eccentricity of lunar orbit
+const double BEIDOU_MOON_INCLINATION = 0.000089803977407e3;  //!< Inclination of lunar orbit to ecliptic plane (5 deg 08 min 43.4 sec) [rad]
+
+const double BEIDOU_SUN_OMEGA = 0.004908229466869e3;  //!< TODO What is this operation in the seconds with T?(281 deg 13 min 15.0 + 6189.03*Т sec) [rad]
+const double BEIDOU_SUN_Q0 = 0.006256583774423e3;     //!< (358 deg 28 min 33.04 sec) [rad]
+const double BEIDOU_SUN_Q1 = 0e3;                     //!< TODO Why is the value greater than 60?(129596579.10 sec) [rad]
+const double BEIDOU_SUN_GM = 0.1325263e12;            //!< Solar gravitational constant [km^3/s^2]
+const double BEIDOU_SUN_SEMI_MAJOR_AXIS = 1.49598e8;  //!< Semi-major axis of solar orbit [km];
+const double BEIDOU_SUN_ECCENTRICITY = 0.016719;      //!< Eccentricity of solar orbit
+
+const double BEIDOU_B2a_CA_FREQ_HZ = FREQ2_BDS;        //!< BEIDOU B2a carrier frequency [Hz]
+const double BEIDOU_B2a_CA_CODE_RATE_HZ = 0.511e6;     //!< BEIDOU L1 C/A code rate [chips/s]
+const double BEIDOU_B2a_CA_CODE_LENGTH_CHIPS = 10230;  //!< BEIDOU B2a C/A code length [chips]
+const double BEIDOU_B2a_CA_CODE_PERIOD = 0.001;        //!< BEIDOU B2a C/A code period [seconds]
+const double BEIDOU_B2a_CA_CHIP_PERIOD = 1.9569e-06;   //!< BEIDOU L1 C/A chip period [seconds]
+const double BEIDOU_B2a_CA_SYMBOL_RATE_SPS = 200;
+const double BEIDOU_B2a_CA_SYMBOL_Length = 600;
+
+const int BEIDOU_CA_NBR_SATS = 24;  // STRING DATA WITHOUT PREAMBLE
+
+/*!
+ * \brief Record of leap seconds definition for GLOT to GPST conversion and vice versa
+ * \details Each entry is defined by an array of 7 elements consisting of yr,month,day,hr,min,sec,utc-gpst
+ * \note Ideally should use leap seconds definitions of rtklib
+ */
+const double BEIDOU_LEAP_SECONDS[19][7] = {
+    {2017, 1, 1, 0, 0, 0, -18},
+    {2015, 7, 1, 0, 0, 0, -17},
+    {2012, 7, 1, 0, 0, 0, -16},
+    {2009, 1, 1, 0, 0, 0, -15},
+    {2006, 1, 1, 0, 0, 0, -14},
+    {1999, 1, 1, 0, 0, 0, -13},
+    {1997, 7, 1, 0, 0, 0, -12},
+    {1996, 1, 1, 0, 0, 0, -11},
+    {1994, 7, 1, 0, 0, 0, -10},
+    {1993, 7, 1, 0, 0, 0, -9},
+    {1992, 7, 1, 0, 0, 0, -8},
+    {1991, 1, 1, 0, 0, 0, -7},
+    {1990, 1, 1, 0, 0, 0, -6},
+    {1988, 1, 1, 0, 0, 0, -5},
+    {1985, 7, 1, 0, 0, 0, -4},
+    {1983, 7, 1, 0, 0, 0, -3},
+    {1982, 7, 1, 0, 0, 0, -2},
+    {1981, 7, 1, 0, 0, 0, -1},
+    {}};
+
+//!< BEIDOU SV's orbital slots PRN = (orbital_slot - 1)
```

Comment:
Is this really for Beidou? or is it a copy/paste error from the GLONASS implementation?